.github/ISSUE_TEMPLATE/vetsapi-sidekiq-ui-access.yaml

```diff
@@ -0,0 +1,99 @@
+        :wave: **_Instructions_**
+        1. Before being granted access, the individual for whom access is requested (the 'target individual') must either be listed on the [Platform Roster](https://vfs.atlassian.net/wiki/spaces/AP/pages/1908834623/Platform+Roster) or have started your [Platform Orientation](https://depo-platform-documentation.scrollhelp.site/getting-started/Platform-Orientation.1877344532.html) and be listed on the [VFS Roster](https://docs.google.com/spreadsheets/d/11dpCJjhs007uC6CWJI6djy3OAvjB8rHB65m0Yj8HXIw/edit?folder=0ALlyxurHpUilUk9PVA#gid=2042046665)
+        2. Change the **Title** to include target individual
+        3. Add the label used by the target individual's team (eg: `BAH-526`)
+        4. Do not remove any of the existing labels.
+        5. Enter required fields and select environments you need access to.
+        6. If Production access is required, please use the respective section to provide proof of E-QIP transmittal. If you have not started the E-QIP process, do not submit this ticket. Ask your Vendor Onboarding Representative to confirm.
+        7. Fill in "Additional Notes" with any information requested by the other steps.
+  - type: input
+    id: requestor-name
+    attributes:
+      label: Your Name
+      description: Please provide the name of the target individual
+      placeholder: Jane Doe
+    validations:
+      required: true
+  - type: input
+    id: email
+    attributes:
+      label: Your GitHub handle (Sidekiq Admin UI access is managed through GitHub)
+      description: Please provide the work GitHub handle of the target individual
+      placeholder: "@username"
+    validations:
+      required: true
+  - type: input
+    id: aws-username
+    attributes:
+      label: Your dsvagovcloud AWS user name (if applicable)
+      description: If you have previously been granted AWS access, list your dsvagovcloud AWS user name
+      placeholder: First.Last
+  - type: input
+    id: team-name
+    attributes:
+      label: Team, Role, and Company of the target individual
+      description: Please provide the name of the team, their role on that team, and the name of the company the target individual works for
+      placeholder: Team Moose, Antler Development, Alces LLC
+    validations:
+      required: true
+  - type: dropdown
+    id: cor-name
+    attributes:
+      label: COR Name
+      description: The name of the Contracting Officer's Representative (COR) covering the target individual.
+      options:
+        - Cecila Lee
+        - Courtney Bethea
+        - Crystal Moultrie
+        - Doris Lin
+        - Vilay Senthep
+        - Eunice Garcia
+        - Delano McVay
+        - Jennifer O'Day
+        - Laurene "Reney" Cook
+        - Rolanda Copeland
+        - Other - please specify in 'Additional Notes'
+    validations:
+      required: true
+  - type: checkboxes
+    attributes:
+      label: Please confirm if you are on the VFS Team Roster or Platform Team Roster
```

Comment:
@little-oddball can VFS teams view that VFS roster? I'm not sure what the permissions are on that.

Reply:
just confirmed that anyone with the link can view 👍

Reply:
oh, nice!